content/synchronization_model.tex

```diff
@@ -19,9 +19,9 @@
 \begin{tabular}{p{0.2\textwidth} | p{0.7\textwidth}}
 {}
 &
-{ Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
+Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
```

Comment:
@abouteiller Looking at the sentence, it seems to me to leave as it is, since we are referring this specific to the diagram.